SampleApp/AppDelegate.swift

```diff
@@ -7,10 +7,10 @@ import UIKit
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-  var window: UIWindow?
-    
-  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey : Any]? = nil) -> Bool {
-    return true
-  }
+    var window: UIWindow?
```

Comment:
I used Appcode formatting which is different from the one used in this project. I'll revert this change.